nautobot/core/ui/object_detail.py

```diff
@@ -1053,6 +1104,33 @@ def get_extra_context(self, context):
         }
 
 
+class _ObjectCommentPanel(ObjectFieldsPanel):
+    """Panel displaying an object's comments as a space-separated panel."""
+
+    def __init__(
+        self,
+        *,
+        label="Comments",
+        section=SectionChoices.LEFT_HALF,
+        weight=Panel.WEIGHT_COMMENTS_PANEL,
+        value_transforms={"comments": [render_markdown, placeholder]},
```

Comment:
pylint is rightly complaining about using a dict as a default arg value. This needs to be `None` and set in the function body if otherwise unspecified.